Error when bundling with Webpack

Hi all

Using webpack bundling Bunyan and I'm seeing the following when the bundling is complete:

```
WARNING in ./~/bunyan/~/dtrace-provider/dtrace-provider.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./build in /home/ray/source/ca.ui.shell.core/node_modules/bunyan/node_modules/dtrace-provider
 @ ./~/bunyan/~/dtrace-provider/dtrace-provider.js 17:22-81
```

Any ideas?

Using version 1.8.0 of bunyan

--- a/README.md
+++ b/README.md
@@ -1374,19 +1374,19 @@ var log = bunyan.createLogger({
 log.info('hi on info');
 ```
 
-## Webpack
-Webpack can work with the same example Browserify above. To do this, we need to make webpack ignore optional files:
+## webpack
+To include bunyan in your webpack bundle you need to tell webpack to
+ignore the optional dependencies that are unavailable in browser environments.
 
-Now tell webpack to ignore files for 
-[optional dependencies](https://webpack.js.org/configuration/module/#module-noparse)
-in your "webpack.config.js":
+Mark the following dependencies as
+[externals](https://webpack.js.org/configuration/externals/) in your webpack
+configuration file to exclude them from the bundle:
+
 ```
 module: {
-    noParse: [/dtrace-provider$/, /safe-json-stringify$/, /mv/],
-    ...
+    externals: ['dtrace-provider', 'fs', 'mv', 'os', 'source-map-support']
 }
 ```
-Now webpack builds.
 
 # Versioning
 

